src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -6837,9 +6872,13 @@ internal Task TdsExecuteSQLBatch(string text, int timeout, SqlNotificationReques
                         _connHandler.CheckEnlistedTransactionBinding();
 
                         stateObj.SetTimeoutSeconds(timeout);
+
                         stateObj.SniContext = SniContext.Snix_Execute;
 
-                        WriteRPCBatchHeaders(stateObj, notificationRequest);
+                        if (_isYukon)
```

Comment:
_isYukon is 2005 and above right? Do we want to add this condition here? .Net Core support is for all supported Sql Server which are 2005 and above.

Reply:
We've left it out before. I'll remove these Yukon checks.

Reply:
Looks like lots of other locations in this file also check for Yukon. I'll remove them in a separate PR.

Reply:
Sounds good